.github/workflows/activation.yml

```diff
@@ -0,0 +1,19 @@
+name: Acquire activation file
+on: [push]
+jobs:
+  activation:
+    name: Request manual activation file 🔑
+    runs-on: ubuntu-latest
+    steps:
+# Request manual activation file
+    - name: Request manual activation file
+      id: getManualLicenseFile
+      uses: webbertakken/unity-request-manual-activation-file@v1.1
+      with:
+        unityVersion: 2019.3.13f1
+# Upload artifact (Unity_v20XX.X.XXXX.alf)
+    - name: Expose as artifact
+      uses: actions/upload-artifact@v1
+      with:
+        name: ${{ steps.getManualLicenseFile.outputs.filePath }}
+        path: ${{ steps.getManualLicenseFile.outputs.filePath }}
```

Comment:
Activation is already part of the following example file:  
https://github.com/Unity-CI/unity-actions-example/blob/main/.github/workflows/main.yml#L12-L41

Reply:
Sorry, I'm just testing some things out on my fork and accidentally created a PR to merge them back into the main project. My bad!

Reply:
ahah no problem 👍 If you want to contribute to the example project afterward, don't hesitate to do so :)